part4/performance-tuning.md

```diff
@@ -28,6 +28,12 @@ If you have generated the tables automatically using your JPA implementation (e.
 -   Sagas
 
     Table 'AssociationValueEntry', columns `associationKey` and `sagaId`,
+    
+    Table 'AssociationValueEntry', columns `sagaId`,
+    
+    Table 'AssociationValueEntry', columns `sagaType` and `sagaId`,
+    
+    Table 'AssociationValueEntry', columns `associationKey` and `associationValue`    
```

Comment:
It is used to find saga's by association key and value through https://github.com/AxonFramework/AxonFramework/blob/e29ea8b6202ab5f58ce24a2158c084e01664af1a/core/src/main/java/org/axonframework/eventhandling/saga/repository/jpa/JpaSagaStore.java#L216 although I assume that by value in that case should be enough.